depends/patches/qt/.gitmodules

```diff
@@ -0,0 +1,50 @@
+[submodule "qtbase"]
```

Comment:
I'm not entirely sure what this does. Does this make qt submodules of our repository?

Reply:
Hi,
No, it does not make qt submodule of bitcoin. The file is just used by qt.pro to determine which qt modules to build, and what their dependencies are. Perhaps, i can rename the file to prevent confusion ?

Reply:
@laanwj I've renamed the files to prevent confusion